warmrestart/warm_restart.cpp

```diff
@@ -1,24 +1,54 @@
 #include <string>
+#include <climits>
 #include "logger.h"
 #include "schema.h"
-#include <climits>
 #include "warm_restart.h"
 
 namespace swss {
 
 const WarmStart::WarmStartStateNameMap WarmStart::warmStartStateNameMap =
 {
-    {INIT,          "init"},
+    {INITIALIZED,   "initialized"},
```

Comment:
If the state name is changed here,  swss-schema.md should be updated too.

Reply:
done